swap_meet/decor.py

```diff
@@ -1,2 +1,27 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Decor(Item):
+    '''
+    override __init__ use super() for id
+    attribute: width=0, length=0
+
+    __str__()
+    returns "An object of type Decor with id <id value>.
+    It takes up a <width value> by <length value> sized space."
+    '''
+    def __init__(self, condition=0, id=None, width=0, length=0):
+        self.condition = condition
+        self.width = width
+        self.length = length
+        super().__init__(id=id, condition=condition)
```

Comment:
If you find yourself initializing an attribute of the same name as one you are passing into your call of `super` its usually a good sign to stop and think about if you are introducing a redundancy by not leveraging your inheritance.